Slim/Menu/BrowseLibrary.pm

```diff
@@ -1555,15 +1555,71 @@ sub _albums {
 			}
 
 			my $extra;
+
+			# Have we got tracks for the year from other albums (ones with a different album year)?
```

Comment:
I understand we are browsing albums for a given year here. Why should we list tracks from albums which have a different year?

Reply:
This assumed my global change to always show all years, not just album years. I'll condition this on the new $hasAlbums prefs toggle.